import-scripts/genie-airflow-set-import-status.sh

```diff
@@ -0,0 +1,20 @@
+IMPORT_STATUS=$1
```

Comment:
can we add a comment here about what this script does (esp. in the context of executed DAG)? I see that the referenced script is also called separately in another script.

Reply:
I left a comment above about this script. Let me know what you think
> I'm not sure if this script is totally needed, but if it doesn't exist and I have the dag call the update_process script directly then I need to set the properties file in the DAG. I can make that change, I just went with the pattern of the other tasks for now, where the properties filepath is set in the bash script.

Reply:
I removed this script and changed the date to call the update_process script directly.